app/src/app/(payload)/custom.scss

```
@@ -98,24 +98,24 @@ body {
+  // The user type is no longer printed on this page (operator decision: the avatar menu states it on
+  // every page, and a signed-in user knows their own account type), so `__head` had one child and every
+  // declaration in it was inert. Deleted rather than trimmed to `margin-bottom`: a flex container with
+  // one child is a container waiting for a second thing to be dropped into it by someone who assumes
+  // the row still exists. The title now carries its own bottom margin.
   &__title {
     // Same size/weight as the frontend page titles (.lp-title / .page-heading h1), via the shared
     // token — so "Manage" sits at the same level in the hierarchy as "Lesson Plans".
     font-size: var(--app-page-title-size);
     font-weight: 700;
-    margin: 0;
+    // Was 0, with the space supplied by `__head`'s margin-bottom; same token, one element fewer.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)